packages/gatsby/src/utils/handle-flags.ts

```diff
@@ -133,17 +143,33 @@ const handleFlags = (
   let message = ``
   //  Create message about what flags are active.
   if (enabledConfigFlags.length > 0) {
-    if (enabledConfigFlags.length - optedInFlags.size > 0) {
+    if (
+      enabledConfigFlags.length - optedInFlags.size - lockedInFlags.size >
+      0
+    ) {
       message = `The following flags are active:`
       enabledConfigFlags.forEach(flag => {
-        if (!optedInFlags.has(flag.name)) {
+        if (!optedInFlags.has(flag.name) && !lockedInFlags.has(flag.name)) {
           message += generateFlagLine(flag)
         }
       })
     }
 
+    if (lockedInFlagsThatAreInConfig.size > 0) {
+      if (message.length > 0) {
+        message += `\n\n`
+      }
+      message += `Some features you configured with flags are now only and not configurable behavior.
+Those flags no longer have any effect and you can remove them from config:`
```

Comment:
halp

Reply:
I know the feeling 😄 Not sure if I can actually help here but maybe something like this:

```suggestion
      message += `Some features you configured with flags are used natively now.
Those flags no longer have any effect and you can remove them from config:`
```